doc/usage/configuration.rst

```diff
@@ -3668,7 +3668,7 @@ and which failures and redirects it ignores.
    .. versionadded:: 4.1
 
    .. versionchanged:: 8.3
-      Setting :confval:`!linkcheck_allowed_redirects` to the empty directory
+      Setting :confval:`!linkcheck_allowed_redirects` to an empty dictionary
```

Comment:
'the empty string', 'the empty dictionary', etc?
```suggestion
      Setting :confval:`!linkcheck_allowed_redirects` to the empty dictionary
```

Reply:
Either is grammatically correct. It depends on the tone you prefer to convey. "The empty dictionary" sounds more like, "there can be only one empty dictionary", whereas "an empty dictionary" sounds more like, "it's all good, there can be any number of empty dictionaries."

Reply:
If there is a preferred style guide for Sphinx, then I'd go with that for consistency.

Reply:
I think that 'the empty directory' is more correct, cf 'the empty set'. But if you think that the indefinite article is friendlier, happy to use that instead.

Reply:
I usually go with the prevailing style guide, but I didn't find one.

https://www.sphinx-doc.org/en/master/internals/contributing.html#contribute-documentation

Oh, and it's merged. We're good.